packages/waku/src/lib/plugins/vite-plugin-fs-router-typegen.ts

```diff
@@ -177,7 +171,9 @@ export const fsRouterTypegenPlugin = (opts: { srcDir: string }): Plugin => {
           }
         }
 
-        let result = `import type { PathsForPages, GetConfigResponse } from 'waku/router';\n\n`;
+        let result = `// deno-fmt-ignore-file
+/* eslint-disable */
```

Comment:
Can we use `prettier-ignore-start`?
https://prettier.io/docs/en/ignore.html#range-ignore

Reply:
<img width="850" alt="image" src="https://github.com/user-attachments/assets/8648de71-5212-4cc6-aedb-2f3112bde3e1" />

when I test it here, it does not work

Reply:
Maybe it doesn't work for js files. Hmm.
Well, I think supporting prettier out of the box is important.
Can we add prettier-ignore for _all_ lines?

Reply:
> Well, I think supporting prettier out of the box is important.

agreed

<img width="599" alt="image" src="https://github.com/user-attachments/assets/d1419a40-baca-45ca-8921-e186eaf638f0" />

it looks like that does technically work when used on each block

I wish there was a better way though

Reply:
this is the generation for 01_template with the latest changes

```ts
// deno-fmt-ignore-file
// biome-ignore format: generated types do not need formatting
/* eslint-disable */
// prettier-ignore
import type { PathsForPages, GetConfigResponse } from 'waku/router';

// prettier-ignore
import type { getConfig as About_getConfig } from './pages/about';
// prettier-ignore
import type { getConfig as Index_getConfig } from './pages/index';

// prettier-ignore
type Page =
| ({ path: '/about' } & GetConfigResponse<typeof About_getConfig>)
| ({ path: '/' } & GetConfigResponse<typeof Index_getConfig>);

// prettier-ignore
declare module 'waku/router' {
  interface RouteConfig {
    paths: PathsForPages<Page>;
  }
  interface CreatePagesConfig {
    pages: Page;
  }
}
```